pkg/services/ngalert/eval/eval.go

```diff
@@ -266,7 +266,7 @@ func getExprRequest(ctx EvaluationContext, data []models.AlertQuery, dsCacheServ
 			if expr.IsDataSource(q.DatasourceUID) {
 				ds = expr.DataSourceModel()
 			} else {
-				ds, err = dsCacheService.GetDatasourceByUID(ctx.Ctx, q.DatasourceUID, ctx.User, true)
+				ds, err = dsCacheService.GetDatasourceByUID(ctx.Ctx, q.DatasourceUID, ctx.User, false /*skipCache*/)
```

Comment:
I don't think you need the comment, but if you want to keep this what about just declaring a `skipCache` variable?

```suggestion
				ds, err = dsCacheService.GetDatasourceByUID(ctx.Ctx, q.DatasourceUID, ctx.User, skipCache)
```

Reply:
static analysis annoys me because it is always false. I think I would prefer the comment.